lib/messages.js

```diff
@@ -39,23 +39,29 @@ function addMessage(opts) {
  * Take array of message objects and locale string and return message content.
  * Use locale if found otherwise defaults to 'en'.
  */
-function getMessage(messages, locale) {
-    var ret = '',
-        found;
-    locale = locale || 'en';
+function getMessage(configuration, locale) {
+    if (!configuration) {
+        return '';
+    }
+
+    // use the translation key if provided
+    if (configuration.translation_key) {
+        return utils.translate(configuration.translation_key, locale);
+    }
+
+    // otherwise, use the configured messages
```

Comment:
Maybe also stick a note in here about `messages` being deprecated?  Either a comment or a warning?